modules/bridge/src/main/ts/ephox/bridge/components/content/ContextForm.ts

```diff
@@ -84,6 +84,7 @@ export interface BaseContextFormSpec<T> extends ContextBarSpec {
   launch?: ContextFormLaunchButtonApi | ContextFormLaunchToggleButtonSpec;
   commands: Array<ContextFormToggleButtonSpec<T> | ContextFormButtonSpec<T>>;
   onInput?: (api: ContextFormInstanceApi<T>) => void;
+  onSetup?: (api: ContextFormInstanceApi<T>) => void;
```

Comment:
This is how to start with what Spocke suggested, look at toolbar buttons or menu items for examples of how `onSetup` is implemented
```suggestion
  onSetup?: (api: ContextFormInstanceApi<T>) => (api: ContextFormInstanceApi<T>) => void;
```

It's so common we have a shared FieldSchema reference for it 😉 
https://github.com/tinymce/tinymce/blob/3fae00c85dd3c664c5b7437b0e6ded5514f6cca4/modules/bridge/src/main/ts/ephox/bridge/core/ComponentSchema.ts#L15